cmd/duffle/bundle_remove.go

```diff
@@ -46,7 +46,7 @@ func newBundleRemoveCmd(w io.Writer) *cobra.Command {
 			return remove.run()
 		},
 	}
-	cmd.Flags().StringVar(&remove.versions, "version", "", "A version or SemVer2 version range")
+	cmd.Flags().StringVarP(&remove.versions, "version", "r", "", "A version or SemVer2 version range")
```

Comment:
So the command should be `duffle bundle rm mybundle -r version` why `-r` as the shortcut for version?

Reply:
I suppose because `-v` is used for verbose output (at least it's supposed to be used that way, Simon pointed out that it's currently not working as intended).

Reply:
I guess the alternative would be not to have a shorthand flag at all?